tendrl/commons/objects/cluster_alert_counters/__init__.py

```diff
@@ -0,0 +1,19 @@
+from tendrl.commons import objects
+
+
+class ClusterAlertCounters(objects.BaseObject):
+    def __init__(
+        self,
+        warn_count=0,
+        integration_id='',
+        *args,
+        **kwargs
+    ):
+        super(ClusterAlertCounters, self).__init__(*args, **kwargs)
+        self.warning_count = warn_count
+        self.integration_id = integration_id
+        self.value = '/clusters/{0}/alert_counters'
+
+    def render(self):
+        self.value = self.value.format(self.integration_id)
```

Comment:
use self.value.format(self.integration_id or NS.tendrl_context.integration_id)

Reply:
No, the problem is while handling gafana  alert server's node-agent going to handle the alert and increment the alert count. so we can't give NS.tendrl_context.integration_id here.  Because it can be any cluster.

Reply:
@GowthamShanmugam This is a common object now. Also,"self.integration_id or NS.tendrl_context.integration_id" is an expression which evaluates to whichever value is present

Make this change